app/models/miq_action.rb

```diff
@@ -627,11 +626,6 @@ def action_delete_snapshots_by_age(action, rec, _inputs)
       return
     end
 
-    if has_ch
-      MiqPolicy.logger.warn("#{log_prefix} has a Consolidate Helper snapshot, no shanpshots will be deleted")
```

Comment:
> no shanpshots

🤣

Reply:
None of those will be deleted either haha